metricbeat/module/jolokia/jmx/_meta/docs.asciidoc

```diff
@@ -0,0 +1,52 @@
+=== jolokia jmx MetricSet
+
+This is the jmx metricset of the module jolokia.
+
+[float]
+=== Features and configuration
+Tested with Jolokia 1.3.4.
+
+Metrics to be collected from each Jolokia instance are defined in the mapping section with an MBean ObjectName and
+an array of Attributes to be requested with Elastic field names under which the return values should be saved.
+
+For example: to get the "Uptime" attribute from the "java.lang:type=Runtime" MBean and map it to something like
+"uptime" (actually "jolokia.jmx.uptime", the prexif is added by beats framework) you have to configure following
+mapping:
+
+```
+- module: jolokia
+  metricsets: ["jmx"]
+  hosts: ["localhost:8778"]
+  namespace: "metrics"
+  jmx.mappings:
+    - mbean: 'java.lang:type=Runtime'
+      attributes:
+        - attr: Uptime
+          field: uptime
+```
+
+In case the underlying attribute is an object (e.g. see HeapMemoryUsage attribute in java.lang:type=Memory) it`s
+structure will be published to Elastic "as is".
+
+It is possible to configure nested metric aliases by using dots in the mapping name (e.g. gc.cms_collection_time). For examples please refer to the
+/jolokia/jmx/test/config.yml.
+
+All metrics from a single mapping will be POSTed to the defined host/port and sent to Elastic as a single event.
+To make it possible to differentiate between metrics from multiple similar applications running on the same host,
+please configure multiple modules.
+
+It is required to set a namespace in the general module config section.
+
+[float]
+=== Limitations
+No authentication against Jolokia is supported yet.
+No wildcards in Jolokia requests supported yet.
+You can get max 30 attributes from the same MBean (see Add function in config.go).
```

Comment:
Is this still a limitation? At list there's not Add function in config.go to check :)